web/track/views.py

```diff
@@ -487,20 +486,22 @@ def verify_account_mobile():
         phone = 'placeholder: ***-***-3333'
         return render_template(generate_path(prefix, "verify-mobile"), phone=phone)
 
-
     def send_pass_reset(user_email, user, prefix):
+        user_name = user.display_name
         if user is not None and user.is_authenticated:
             password_reset_serial = URLSafeTimedSerializer(api_config.super_secret_key)
             password_reset_url = url_for(prefix + '_new_password',
                                          token=password_reset_serial.dumps(user_email, salt=api_config.super_secret_salt),
                                          _external=True
                                          )
-            return render_template(generate_path(prefix, "reset-email-temp"), password_reset_url=password_reset_url)
-            # response = notifications_client.send_email_notification(
-            #     email_address='email_address',
-            #     template_id='6e3368a7-0d75-47b1-b4b2-878234e554c9'we
-            # )
-            # General Notification
+            response = notifications_client.send_email_notification(
+                email_address=user_email,
```

Comment:
Can we just get the user_email from the user obj being passed into this function instead of passing redundant params?